### PR TITLE
Feature/issue 13/build stock tracker

### PR DESCRIPTION
**Pull request by joquendo**

### CODE REVIEW

**Comment by joquendo** [comment]
PR using AlphaVantage and React Plotly.js. I added the API key locally to not expose the API key. The page will load stock quotes' prices for the last 100 days and plot them on a graph. It will look like the following image:
Screen Shot 2021-06-16 at 9 16 34 AM

File: src/setupTests.tsx [hunk 1]
```diff
@@ -3,3 +3,5 @@
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
 import "@testing-library/jest-dom";
+
+window.URL.createObjectURL = function() {};
```
Comment (joquendo) [inline src/setupTests.tsx:7]: Added to fix an issue with Jest and React-plotly.js

File: src/App.test.tsx [hunk 2]
```diff
@@ -4,6 +4,6 @@ import App from "./App";
 
 test("Stock App in document", () => {
   render(<App />);
-  const heading = screen.getByText(/Stock App/i);
+  const heading = screen.getByText(/Stock App/i, {selector: '.logo'});
```
Comment (joquendo) [inline src/App.test.tsx:7]: Fix Jest test by updating selector to match what is rendered.

**Comment by ShelbsH** [comment]
If this is a modern React app, I don't see the need to use class components anymore.

There's also 4 spaces in this file, while another file is using two spaces, which is inconsistent. There should be a standard on what everyone would agree on in code formatting.

**Review by Schraut — APPROVED** [review]
It works @joquendo. I had to create a .env file, created my own API key and placed it into the .env file, and then I had to install ploty with  "npm install plotly.js". After that, everything worked just fine.

**Pull request merged**